src/EditorManager.ts

```diff
@@ -41,19 +41,19 @@ export interface ShowEditorDocumentOptions {
 
 type PersistedEditors = { [key: string]: string };
 
-export class CosmosEditorManager {
-    private fileMap: { [key: string]: ICosmosEditor } = {};
+export class EditorManager {
```

Comment:
Can you please revert stuff like this that's just an internal engineering change so that the diff is much smaller? Branding is about strings visible to the user (not internal engineering changes) and seemingly simple changes can cause nasty bugs only discoverable once you release, so I want to be extra careful.

Plus - if you submit a separate PR with _just_ the engineering name changes, I can approve that very quickly since I know VS Code did most of the changes automatically for us anyways (and CI should catch any problems)